concent_api/core/tests/test_core_views.py

```diff
@@ -558,6 +556,37 @@ def test_send_with_empty_data_should_return_http_400_error(self):
 
         self.assertEqual(response.status_code, 400)
 
+    def test_send_should_return_http_400_if_task_to_compute_younger_than_report_computed(self):
+
+        deserialized_task_to_compute = self._get_deserialized_task_to_compute(
+            task_id     = '1',
+            subtask_id  = '3',
+            deadline    = get_current_utc_timestamp() + (60 * 37),
+            provider_public_key  = self.PROVIDER_PUBLIC_KEY,
+            requestor_public_key = self.REQUESTOR_PUBLIC_KEY,
+        )
+        with freeze_time("2017-11-17 10:00:00"):
+            report_computed_task = message.tasks.ReportComputedTask(
+                task_to_compute = deserialized_task_to_compute
+            )
+            force_report_computed_task = message.ForceReportComputedTask(
+                report_computed_task = report_computed_task
+            )
+
+            response = self.client.post(
+                reverse('core:send'),
+                data = dump(
+                    force_report_computed_task,
+                    self.PROVIDER_PRIVATE_KEY,
+                    CONCENT_PUBLIC_KEY
+                ),
+                content_type                        = 'application/octet-stream',
+                HTTP_CONCENT_CLIENT_PUBLIC_KEY      = self._get_encoded_provider_public_key(),
+                HTTP_CONCENT_OTHER_PARTY_PUBLIC_KEY = self._get_encoded_requestor_public_key(),
+            )
+
+        self.assertEqual(response.status_code, 400)
```

Comment:
I think I have already seen a method for this. Sth like `self._assert_400`.

Reply:
`self._test_400_response()` Done